orga/tests/integration/components/campaign/list_test.js

```diff
@@ -347,4 +349,189 @@ module('Integration | Component | Campaign::List', function (hooks) {
+  @campaigns={{this.campaigns}}
+  @onFilter={{this.noop}}
+  @onClickCampaign={{this.noop}}
+  @listOnlyCampaignsOfCurrentUser={{true}}
+/>`,
+      );
+
+      const firstCampaignCheckbox = screen.getAllByRole('checkbox')[1];
+      await click(firstCampaignCheckbox);
+
+      await fillByLabel(this.intl.t('pages.campaigns-list.filter.by-name'), '1');
+
+      // then
+      assert.false(firstCampaignCheckbox.checked);
+    });
+
+    test('should reset selected campaigns when resetting filters', async function (assert) {
+      // when
+      const screen = await render(
+        hbs`<Campaign::List
+  @campaigns={{this.campaigns}}
+  @onFilter={{this.noop}}
+  @onClickCampaign={{this.noop}}
+  @listOnlyCampaignsOfCurrentUser={{true}}
+  @nameFilter='1'
+  @onClear={{this.noop}}
+/>`,
+      );
+
+      const firstCampaignCheckbox = screen.getAllByRole('checkbox')[1];
+      await click(firstCampaignCheckbox);
+
+      const resetButton = await screen.findByRole('button', {
+        name: this.intl.t('common.filters.actions.clear'),
+      });
+      await click(resetButton);
+
+      // then
+      assert.false(firstCampaignCheckbox.checked);
+    });
+
+    test('should delete campaigns', async function (assert) {
+      class CurrentUserStub extends Service {
+        organization = { id: 1 };
+      }
+      this.owner.register('service:current-user', CurrentUserStub);
+      const deleteStub = sinon.stub();
+      class StoreStub extends Service {
+        adapterFor = () => ({ delete: deleteStub });
+      }
+      // We have to do this because of previous usage of lookup
+      this.owner.unregister('service:store');
+      this.owner.register('service:store', StoreStub);
+      // when
+      const onDeleteCampaignsStub = sinon.stub();
+      this.set('onDeleteCampaigns', onDeleteCampaignsStub);
+
+      const screen = await render(
+        hbs`<Campaign::List
+  @campaigns={{this.campaigns}}
+  @onFilter={{this.noop}}
+  @onClickCampaign={{this.noop}}
+  @listOnlyCampaignsOfCurrentUser={{true}}
+  @onDeleteCampaigns={{this.onDeleteCampaigns}}
+/>`,
+      );
+
+      await click(screen.getAllByRole('checkbox')[1]);
+      await click(screen.getAllByRole('checkbox')[2]);
+
+      const deleteButton = await screen.findByRole('button', {
+        name: this.intl.t('pages.campaigns-list.action-bar.delete-button'),
+      });
+      await click(deleteButton);
+
+      await screen.findByRole('dialog');
+
+      const allowMultipleDeletionCheckbox = await screen.findByRole('checkbox', {
+        name: this.intl.t('components.ui.deletion-modal.confirmation-checkbox', { count: 2 }),
+      });
+      await click(allowMultipleDeletionCheckbox);
+
+      const confirmationButton = await screen.findByRole('button', {
+        name: this.intl.t('components.ui.deletion-modal.confirm-deletion'),
+      });
+      await click(confirmationButton);
+
+      //then
+      assert.ok(onDeleteCampaignsStub.called);
+      assert.ok(deleteStub.called);
```

Comment:
**To do :** ajouter un calledWith() pour vérifier qu'il a bien été appelé avec les bons ids